src/gwc/src/main/java/org/geoserver/gwc/FakeHttpServletRequest.java

```diff
@@ -62,6 +66,13 @@ public FakeHttpServletRequest(
         this.parameterMap = parameterMap;
         this.cookies = cookies;
         this.workspace = workspace;
+        // grab the original request from Spring to forward security related attributes
+        // such as requests host, ports and headers
+        this.original =
+                Optional.ofNullable(
+                                (ServletRequestAttributes)
+                                        RequestContextHolder.getRequestAttributes())
+                        .map(atts -> atts.getRequest());
```

Comment:
Not sure if you notice, a method reference can be used here:
```
        this.original =
                Optional.ofNullable(
                                (ServletRequestAttributes)
                                        RequestContextHolder.getRequestAttributes())
                        .map(ServletRequestAttributes::getRequest);
```

Reply:
I did not think about it, but not sure replacing 26 chars with 37 to do the same thing is an improvement ;-)

Reply:
:dancer: